.github/workflows/ci_flutter.yml

```diff
@@ -0,0 +1,53 @@
+name: Build flutter example
+
+on:
+  workflow_call:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  flutter-build-example:
+    # The Android dependencies are built automatically when running the flutter build,
```

Comment:
The reason we don't need the Android libraries here is that for Android we have a dynamic library that is loaded at runtime, while for iOS we have a static library that needs to exist during the linking process. The android example will build but fail when you try to run the example